### ISSUE TITLE
[feature] ability to reverse `moon ci` task logic

### ISSUE THREAD

**Issue by shellscape** [moonrepo/moon#1047]
**Is your feature request related to a problem? Please describe.**

By default `moon ci` runs _everything_ and it's up to us to annotate everything we don't want it to run. We've got a lot of tasks we don't want it running, and conversely only have a few that we want it to run.

**Describe the solution you'd like**

I'd like the ability to opt-in to `moon ci` running a task, rather than opt-out, as the opt-out is burdensome in a few repos I'm working in.

**Describe alternatives you've considered**

Aren't really any other ways to get around this one at present.

**Additional context**

In the repos I'm using moon, fine grained CI control and setup is preferred over letting `moon ci` run all tasks. Linting, building, etc are preferred in separate steps with separate reporting in Github Actions. The DX for that is viewed as better than having a single task step output the results for steps which are seen as separate, incremental, blocking steps. e.g. We perform linting first. Linting blocks Building. Building blocks Deployment. A failure in the previous step prevents the following steps from executing. While it might not be everyone's cup of tea, it's what we like and serves us well. A single step with output for all-the-things is hard to read, and hard to parse in a repo wit 20+ projects. We also lint the entire repo and cache eslintcache in the workflow to improve speed. We do this because it's a safer bet.

Our end goal is to have `moon ci` run only our deployment step, as that's the only part of our task system that requires knowing what the affected files are.

**Comment by milesj** [comment]
Not opposed to it, will look into 👍

**Comment by milesj** [comment]
@shellscape This should do it https://github.com/moonrepo/moon/pull/1056

**Comment by milesj** [comment]
Can pass targets now in v1.14!

**Comment by shellscape** [comment]
Huge!
